### ISSUE TITLE
MPX instructions with 0x67 prefix

### ISSUE THREAD

**Issue by Gumix** [intelxed/xed#57]
Hi!

Currently xed allows to change the address size of bndmov by 0x67 prefix:

>./obj/examples/xed -mpx -64 -d 66 0f 1b 08
660F1B08
ICLASS: BNDMOV   CATEGORY: MPX   EXTENSION: MPX  IFORM: BNDMOV_MEMdq_BND   ISA_SET: MPX
SHORT: bndmov xmmword ptr [rax], bnd1

>./obj/examples/xed -mpx -64 -d 67 66 0f 1b 08
67660F1B08
ICLASS: BNDMOV   CATEGORY: MPX   EXTENSION: MPX  IFORM: BNDMOV_MEMdq_BND   ISA_SET: MPX
SHORT: bndmov xmmword ptr [eax], bnd1

However [objdump ignores it](https://sourceware.org/bugzilla/show_bug.cgi?id=18771). I didn't find anything regarding this in SDM, so not 100% sure which behaviour is correct.

**Comment by markcharney** [comment]
Another fun one... Last looked at this many years ago. I suspect that objdump is correct in 64b mode. There is some history here that I need to sort out first. If objdump is correct and 67 is ignored, then this might be a bit annoying since ignoring 67 prefixes is very nonstandard.

In 32b mode, the SDM has some verbiage about CS.D in the list of exceptions for the BNDMOV instr: 

If 67H prefix is not used and CS.D=0.
If 67H prefix is used and CS.D=1.

The SDM is trying to say only 32b addressing is supported outside of 64b mode. Hopefully objdump got that part right too.

**Comment by Gumix** [comment]
Oh, there is "Table 17-3. Effective Address Size of Intel MPX Instructions with 67H Prefix" in SDM.

**Comment by markcharney** [comment]
yeah. That's the target behavior. It is proving difficult to fix this particular issue.  I'm working on it. Might take a while.
